man/Makefile

```diff
@@ -51,21 +51,24 @@ pdf_pages  = $(addsuffix .pdf,$(man_pages))
 #
 # though rst2man doesn't accept it. docutils must be fixed.
 #
-all: $(man_pages) $(html_pages) # $(pdf_pages)
+all: $(man_pages) $(html_pages) $(pdf_pages)
```

Comment:
(Out of scope comments are really welcome if you have time :-)

man/Makefile is for us, people who are writing the pages.
For users, targets for generating files are in Makefile.am of top level.
For Unix-like platform, I think providing .1 and .7 man pages is enough.
However, for Windows, I'm not sure. If man command is not avaiable, people may want alternatives.
Such platforms may have *.mak files. So we should add "pdf" or something other targets to the .mak files. Putting html files derived from ctags.1.rst.in is another issue.

About man/Makefile, change as you wants. There is not impact on our users.
About *.mak files for platforms other than Unix-like OS, change as you wants. 
About Makefile.am, make a pull request or open an issue. I would like to work together.

Any way, adding more targets to man/Makfile is good idea. I will add them.